File: sample/src/main/java/contacts/sample/cheatsheet/accounts/java/MoveRawContactsAcrossAccountsActivity.java
package contacts.sample.cheatsheet.accounts.java;

import android.accounts.Account;
import android.app.Activity;

import contacts.core.ContactsFactory;
import contacts.core.accounts.MoveRawContactsAcrossAccounts;
import contacts.core.entities.RawContact;

public class MoveRawContactsAcrossAccountsActivity extends Activity {

    MoveRawContactsAcrossAccounts.Result moveRawContactToAccount(
            RawContact rawContact, Account account
    ) {
        return ContactsFactory.create(this)
                .accounts()
                .move()
                .rawContactsTo(account, rawContact)
                .commit();
    }
}

File: sample/src/main/java/contacts/sample/cheatsheet/accounts/kotlin/MoveRawContactsAcrossAccountsActivity.kt
package contacts.sample.cheatsheet.accounts.kotlin

import android.accounts.Account
import android.app.Activity
import contacts.core.Contacts
import contacts.core.accounts.MoveRawContactsAcrossAccounts
import contacts.core.entities.RawContact

class MoveRawContactsAcrossAccountsActivity : Activity() {

    fun moveRawContactToAccount(
        rawContact: RawContact, account: Account
    ): MoveRawContactsAcrossAccounts.Result = Contacts(this)
        .accounts()
        .move()
        .rawContactsTo(account, rawContact)
        .commit()
}